doc/api-extensions.md

```diff
@@ -2480,3 +2480,7 @@ When set to `on`, if the host has guest attachment enabled, the guest can reques
 
 This adds entity type metadata to `GET /1.0/metadata/configuration`.
 The entity type metadata is a JSON object under the `entities` key.
+
+## `limits_cpu_pin_strategy`
```

Comment:
Lets call this `vm_limits_cpu_ping_strategy`

Reply:
Just making sure a copy-n-pasto doesn't happen: `s/ping/pin/` :)